apps/web/src/pages/subscribers/SubscribersListPage.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import { Badge, ActionIcon, useMantineTheme } from '@mantine/core';
+import { Link, useNavigate } from 'react-router-dom';
+import { ColumnWithStrictAccessor } from 'react-table';
+import styled from '@emotion/styled';
+import { format } from 'date-fns';
+import { useSubscribers } from '../../api/hooks/use-subscribers';
+import PageMeta from '../../components/layout/components/PageMeta';
+import PageHeader from '../../components/layout/components/PageHeader';
+import PageContainer from '../../components/layout/components/PageContainer';
+import { Table, Text } from '../../design-system';
+import { Tooltip } from '../../design-system';
+import { Data } from '../../design-system/table/Table';
+import { useEnvController } from '../../store/use-env-controller';
+
+function SubscribersList() {
+  const { readonly } = useEnvController();
+  const [page, setPage] = useState<number>(0);
+  const { subscibers, loading: isLoading, totalCount, pageSize } = useSubscribers(page);
+  const theme = useMantineTheme();
+  const navigate = useNavigate();
+
+  function handleTableChange(pageIndex) {
+    setPage(pageIndex);
+  }
+
+  const columns: ColumnWithStrictAccessor<Data>[] = [
```

Comment:
this variable could be moved out of this component body, id doesn't need to be recreated on every render

Reply:
fixed!